cpp/turbodbc_arrow/Library/src/arrow_result_set.cpp

```diff
@@ -81,8 +75,8 @@ namespace {
     epoch.tm_year = 70;
     epoch.tm_mday = 1;
 
-    // Milliseconds since the epoch
-    return lrint(difftime(mktime(&date), mktime(&epoch)) * 1000);
+    // days since the epoch
+    return lrint(difftime(mktime(&date), mktime(&epoch)) / 86400);
```

Comment:
Take note of what was required for msvc compatibility https://github.com/apache/arrow/blob/master/cpp/src/arrow/python/util/datetime.h#L35